### DISCUSSION TITLE
Is it possible to send float messges

### DISCUSSION THREAD

**Discussion by rolandkarlson** [orchetect/swift-osc#28] [General]
Hi thanks for great library, Is it possible to send float messages?

**Comment by orchetect** [comment]
An OSC message can contain one or more floating-point values.

**Reply by rolandkarlson** [reply]
if i send OSCMessage("/button19", values: [ 0.5 ]) then the 0.5 is truncated to 0. i think this is a bug

**Reply by marko-krueger** [reply]
Sending floats works over here... are you sure OSCKit is truncating it, not the receiver maybe?

**Reply by orchetect** [reply]
Float and Double are different types. Try sending Float(0.5) and then Double(0.5) and see which works for you.
